include/zephyr/ipc/ipc_service.h

```diff
@@ -21,105 +21,12 @@ extern "C" {
  * @{
  * @}
  */
+
 /**
  * @brief IPC Service API
  * @defgroup ipc_service_api IPC service APIs
  * @ingroup ipc
  * @{
- *
- * Some terminology:
- *
```

Comment:
Why are we removing this? I think it's a really handy guide to have here TBH.

Maybe we can hide that using `@cond INTERNAL_HIDDEN` ?

Reply:
I've just changed the location. Everything is in the documentation file. Having the doc in two places equals more maintenance when something changes. Do you want to keep this?
I took an approach from other subsystems - e.g logging.

Reply:
Honestly I don't see the nomenclature changing anytime soon. Please keep it adding the `INTERNAL_HIDDEN` cond